tests/acceptance/guid-node/registrations-test.ts

```diff
@@ -57,10 +57,164 @@ module('Acceptance | guid-node/registrations', hooks => {
+        );
+
+        assert.dom('[data-test-node-card]').exists({ count: 1 });
+
+        assert.dom('[data-test-node-card-heading]').includesText(node.title);
+
+        assert.dom('[data-test-node-card-body]').includesText(registrationSchemaName);
+
+        assert.dom('[data-test-node-card-body]').includesText(registrationTitle);
+
+        await untrackedClick('[data-test-registrations-container] a[href="#drafts"]');
+
+        assert.dom('[data-test-registrations-pane]').isNotVisible();
+        assert.dom('[data-test-draft-registrations-pane]').isVisible();
+
+        assert.dom('[data-test-draft-registrations-pane]').includesText(
+            'There are no draft registrations of this project.',
+        );
+    });
+
+    test('logged in write-permission, 1 draft registration', async assert => {
+        const initiator = server.create('user', 'loggedIn');
+
+        const node = server.create('node', {
+            id: 'decaf',
+            currentUserPermissions: [Permission.Write, Permission.Read],
+        });
+
+        server.loadFixtures('schema-blocks');
+        server.loadFixtures('registration-schemas');
+        server.loadFixtures('registration-providers');
+        server.loadFixtures('licenses');
+
+        const registrationSchema = server.schema.registrationSchemas.all().models.filter(schema =>
+            schema.name === 'Prereg Challenge')[0];
+
+        const registrationMetadata = {
+            q1: { comments: [], value: 'Registration Title', extra: [] },
+        };
+
+        draftRegisterNode(server, node, { initiator, registrationSchema, registrationMetadata });
+
+        const url = `/${node.id}/registrations`;
+
+        await visit(url);
+
+        assert.equal(currentURL(), url, `We are on ${url}`);
+
+        assert.dom('[data-test-new-registration-button]').doesNotExist(
+            'write-contributors cannot create new draft registration',
+        );
+
+        assert.dom('[data-test-registrations-pane]').includesText(
+            'There have been no completed registrations of this project.',
+        );
+
+        await untrackedClick('[data-test-registrations-container] a[href="#drafts"]');
+
+        assert.dom('[data-test-draft-registrations-pane]').isVisible();
+
+        assert.dom('[data-test-draft-registrations-pane]').doesNotIncludeText(
+            'There are no draft registrations of this project.',
+        );
+
+        assert.dom('[data-test-draft-registration-card]').exists({ count: 1 });
+
+        assert.dom('[data-test-draft-registration-card-title]').includesText(
+            'Prereg Challenge',
+        );
+
+        assert.dom('[data-test-draft-registration-card-progress-bar]').exists({ count: 1 });
+
+        const progressBarElement =
+            document.querySelector('[data-test-draft-registration-card-progress-bar] .progress-bar') as HTMLElement;
+
+        assert.ok(
+            parseFloat(progressBarElement.style.width ? progressBarElement.style.width : '') > 0,
```

Comment:
Doesn't this give us same info as 👇?
```javascript
  assert.dom('[data-test-draft-registration-card-progress-bar]').isVisible();
```

Reply:
I would think this does the same thing, but I kept that test the same as it is in the other tests since it seemed very deliberate. I checked the history of the file and I'm not too sure why this test was written the way it was though

Reply:
Oh, there is an assertion above it is checking to see if it exists, this assertion is making sure the progress bar is working 😸